### PR TITLE
Feature/fix responsiveness

### PR DESCRIPTION
**Pull request by JamesSCBD**

### CODE REVIEW

File: store/coreui.js [hunk 1]
```diff
@@ -3,31 +3,44 @@ const $state = () => ({
...
+
+  toggleSidebarMobile(state) {
+    const sidebarClosed = [ false, 'responsive' ].includes(state.sidebarShow);
+    state.sidebarShow   = sidebarClosed ? true : 'responsive';
   },
 
-  isShow({ sideBar }) {
-    return sideBar.show;
+  set(state, [ variable, value ]) {
+    state[variable] = value;
```
Comment (sbilodeau) [inline store/coreui.js:24]: please don't do that. you cannot allow user to pass arbitrary variable.

Comment (sbilodeau) [reply store/coreui.js:24]: vuex will throw an exception if you try to set an variable on state thet is not defined at boot

File: store/coreui.js [hunk 2]
```diff
@@ -3,31 +3,44 @@ const $state = () => ({
...
+  toggleSidebarDesktop({ commit }) {
+    commit('toggleSidebarDesktop');
+  },
+  toggleSidebarMobile({ commit }) {
+    commit('toggleSidebarMobile');
   },
-  sideBarMinToggle(state) {
-    state.sideBar.min = !state.sideBar.min;
+  setSidebar({ commit }, [ variable, value ]) {
+    commit('set', [ variable, value ]);
```
Comment (sbilodeau) [inline store/coreui.js:37]: same as for line 23
please don't do that. you cannot allow user to pass arbitrary variables.

File: components/kronos/sidebar/index.vue [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
     <CSidebarBrand
       :img-full="{ width: 180, height: 46, alt: 'Logo', src: '/k-logo-full.svg'}"
       :img-minimized="{ width: 50, height: 55, alt: 'Logo', src: '/k-logo-mobi.svg'}"
       :wrapped-in-link="{ href: 'https://github.com/scbd/kronos-web', target: '_blank'}"
     />
     <Nav />
-    <CSidebarMinimizer class="d-md-down-none" @click.native="toggleMin()" />
+    <CSidebarMinimizer
+      class="d-md-down-none"
+      @click.native="setSidebar(['sidebarMinimize', !minimize])"
```
Comment (sbilodeau) [inline components/kronos/sidebar/index.vue:16]: call this action `toggleSideBarMinimized()`. no need to pass the value as the state knows it.

File: components/kronos/sidebar/index.vue [hunk 4]
```diff
@@ -1,17 +1,25 @@
 <template>
-  <CSidebar fixed :minimize="isMin" :show="isShow" @update:show="toggleShow()">
+  <CSidebar
+    unfoldable
+    :minimize="minimize"
+    :show="show"
+    @update:show="(value) => setSidebar(['sidebarShow', value])"
```
Comment (sbilodeau) [inline components/kronos/sidebar/index.vue:6]: call this action toggleSideBarVisible(). no need to pass the value as the state knows it.

File: components/kronos/sidebar/index.vue (hunk 3 above)
Comment (JamesSCBD) [reply components/kronos/sidebar/index.vue]: Done! Create new action with this name `toggleSideBarMinimized()`

File: components/kronos/sidebar/index.vue (hunk 4 above)
Comment (JamesSCBD) [reply components/kronos/sidebar/index.vue]: Done! Create new action with this name `toggleSideBarVisible()`

File: store/coreui.js (hunk 1 above)
Comment (JamesSCBD) [reply store/coreui.js]: Removed such `mutation`

File: store/coreui.js (hunk 2 above)
Comment (JamesSCBD) [reply store/coreui.js]: Removed such `mutation`

**Pull request merged**